src/ts-jest-transformer.spec.ts

```diff
@@ -117,7 +117,7 @@ Array [
 
   it('should return stringified version of file', () => {
     config.shouldStringifyContent.mockImplementation(() => true)
-    expect(process()).toMatchInlineSnapshot(`"ts:module.exports=\\"export default \\\\\\"foo\\\\\\"\\""`)
+    expect(process()).toMatchInlineSnapshot(`"module.exports=\\"export default \\\\\\"foo\\\\\\"\\""`)
```

Comment:
this was actually a tiny "bug", files which should be transformed into modules returning their content were then going thru typescript, which was not needed at all, but not failing of course